.storybook/components/Docs/Guidelines/Theming.mdx

```diff
@@ -153,45 +153,24 @@ npx eds-import-from-figma-api file <file_id> --token <personal_access_token>
 
 When you have your own custom theme, you can use the tokens provided in `app-tailwind-theme.config.json` to do advanced tailwind configuration. This file contains all the tokens in JSON format, mapped to the literal values in your local theme.
 
-You can use similar import values to what is in `@chanzuckerberg/eds/tailwind.config.ts` in your local tailwind configuration file:
+EDS provides a utility method to pull in the theme config and generate the proper tailwind 3.x config object.
+
+**NOTE**: Tailwind 4.x support to come in a future release of EDS.
 
 ```ts
 // in your tailwind.config.ts file
 import type { Config } from 'tailwindcss';
-import baseConfig from '@chanzuckerberg/eds/tailwind.config';
+import {applyTailwindConfig} from '@chanzuckerberg/eds/tailwind.config';
```

Comment:
Previously, one would import the config from EDS, and mix/match it with the local config. Now, the base config can come thru this method. The easy path is to use it directly, but doesn't prevent the previous behavior